### PR TITLE
Get the qgisserver plugin from geomapfish_qgisserver repo

### PR DESCRIPTION
**Pull request by sbrunner**
https://github.com/camptocamp/geomapfish_qgisserver

### CODE REVIEW

File: docker/qgisserver/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM camptocamp/qgis-server:3
+LABEL maintainer Camptocamp "info@camptocamp.com"
+
+COPY * /var/www/plugins/geomapfish_qgisserver/
```
Comment (pvalsecc) [inline docker/qgisserver/Dockerfile]: How is the plugin enabled in QGIS?

**Comment by pvalsecc** [comment]
Shouldn't you change the push target to add this image? Add the target to build it? Change the docker-compsoe to use it?

File: docker/qgisserver/Dockerfile (hunk 1 above)
Comment (sbrunner) [reply docker/qgisserver/Dockerfile]: As I remember QGIS take everything that's in this folder

**Comment by sbrunner** [comment]
The goal is to add what's in the previous repository with minimal changes, the integration with the CI will comm later :-)

**Pull request merged**